doc/server.md

```diff
@@ -12,6 +12,8 @@ Key                             | Type      | Default   | API extension
 :--                             | :---      | :------   | :------------            | :----------
 backups.compression\_algorithm  | string    | gzip      | backup\_compression      | Compression algorithm to use for new images (bzip2, gzip, lzma, xz or none)
 candid.api.url                  | string    | -         | candid\_authentication   | URL of the the external authentication endpoint using Candid
+candid.expiry                   | integer   | 3600      | candid\_config           | Candid macaroon expiry
+candid.domains                  | string    | -         | candid\_config           | Allowed Candid domains
```

Comment:
Separator should probably be documented too (comma separator with the parser stripping spaces would be consistent with others)